client/src/rpb/simple_view.js

```diff
@@ -46,105 +47,99 @@ class SimpleView extends React.Component {
     } = this.props;
     return (
       <div> 
-        <LabeledBox
-          label={ <TextMaker text_key="blue_text_table_controls" /> }
-          content={
-            <div className="rpb-config rpb-simple row">
-              <div className="col-md-6">
-                <fieldset className="rpb-config-item col-selection simple">
-                  <legend className="rpb-config-header"> <TextMaker text_key="select_columns" /> </legend>
-                  <SelectList 
-                    items={_.map(all_data_columns, obj => ({id: obj.nick, display: obj.fully_qualified_name }) ) }
-                    selected={ _.map(columns,'nick') }
-                    is_multi_select={true}
-                    onSelect={id=> on_toggle_col_nick(id) }
-                  />
-                </fieldset>
-              </div>
-              <div className="col-md-6">
-                { subject === Gov && 
-                  <div className="rpb-config-item">
-                    <label 
-                      className="rpb-config-header" 
-                    > 
-                      <TextMaker text_key="show_dept_breakout" />
-                      <input 
-                        type="checkbox"
-                        disabled={subject!==Gov}
-                        checked={deptBreakoutMode}
-                        onChange={on_toggle_deptBreakout}
-                        style={{ marginLeft: '15px' }}
-                      />
+        <LabeledBox label={<TextMaker text_key="blue_text_table_controls" />}>
```

Comment:
These text keys are sort of bizarrely named since the text isn't even blu (any more)